command/refresh.go

```diff
@@ -21,8 +21,10 @@ func (c *RefreshCommand) Run(args []string) int {
 		return 1
 	}
 
+	var statePath string
+
 	cmdFlags := c.Meta.extendedFlagSet("refresh")
-	cmdFlags.StringVar(&c.Meta.statePath, "state", DefaultStateFilename, "path")
+	cmdFlags.StringVar(&statePath, "state", "", "path")
```

Comment:
If you leave the first argument as-is, you can omit the later if statement.

```suggestion
	cmdFlags.StringVar(&c.Meta.statePath, "state", DefaultStateFilename, "path")
```

Reply:
Hi @mildwonkey ,
Thanks for reply.

Should I set `DefaultStateFilename` (in 31:command.go) to ""?
or set this line to `cmdFlags.StringVar(&c.Meta.statePath, "state", "", "path")` ?

Reply:
That's a great question and I'm so sorry the suggestion in my comment wasn't quite right - great catch, thank you so much. I updated the suggestion; your second idea option is correct.